static/app/views/projectInstall/messagingIntegrationAlertRule.tsx

```diff
@@ -38,17 +38,6 @@ export default function MessagingIntegrationAlertRule({
     [providersToIntegrations, provider]
   );
 
-  useEffect(() => {
-    const providerKeys = Object.keys(providersToIntegrations);
-    if (providerKeys.length > 0) {
-      const firstProvider = providerKeys[0];
-      setProvider(firstProvider);
-
-      const firstIntegration = providersToIntegrations[firstProvider][0];
-      setIntegration(firstIntegration);
-    }
-  }, [providersToIntegrations, setProvider, setIntegration]);
-
```

Comment:
these initial values are still being set inside of the `useCreateNotificationAction` hook, I just forgot to delete this `useEffect` after I created the hook 😭